pyanaconda/ui/cockpit/anaconda-webui/test/run-test

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+set -ex
+
+clean_up () {
+    ARG=$?
```

Comment:
I think you should do `set +e` here, so that the virsh cleanups aren't skipped if the pkill or first virsh command fail.